about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+    <script src="https://kit.fontawesome.com/f25b74c76b.js" crossorigin="anonymous"></script>
+    <title>capstone-project</title>
+</head>
+<body>
+  <header id="dsktop-header">
+    <nav id="nav1" title="first-navigation">
+      <ul>
+        <li><a class="fa-brands fa-facebook-f"></a></li>
+        <li><a class="fa-brands fa-twitter"></a></li>
+        <li><a href="#" class="nav1item">English</a></li>
+        <li><a href="#" class="nav1item">My Page</a></li>
+        <li><a href="#" class="nav1item">Logout</a></li>
+      </ul>
+    </nav>
+    
+    <nav id="nav2" title="second-navigation"> 
+      <a class="main-logo" href="#about"><img src="images/icecream.png" alt=""></a> 
+      <a class="nav2Item" href="./index.html">Home</a>
+      <a class="nav2Item" href="./index.html">Packages</a>
+      <a class="nav2Item" href="#join">Join</a>
+      <a class="nav2Item" href="#sponsor">Sponsor</a>
+      <a class="nav2Item" href="#news">Recipes</a>
+      <a class="mainItem" href="./index.html">workshops</a> 
+    </nav>   
+</header>
+
+<header id="mobile-header">
+  <nav class="mobile-nav"> 
+      <ul onclick="menuPopup()" id="navList" title="navigation-list">
+        <li><a class="navItem" href="./index.html">Home</a></li>
+        <li><a class="navItem" href="./index.html">Our Packages</a></li>
+        <li><a class="navItem" href="#join">Join</a></li>
+        <li><a class="navItem" href="#sponsor">Sponsor</a></li>
+        <li><a class="navItem" href="#news">Recipes</a></li>
+        <li><a class="navItem navItem-main" href="#about">workshops</a></li>
+      </ul> 
+  </nav>   
+  <nav>
+    <div onclick="menuPopup()" id="hamburger">
+      <span class="bar"></span>
+      <span class="bar"></span>
+      <span class="bar"></span>
+    </div> 
+    <div onclick="menuPopup()" id="closeButton"> &times; </div>
+  </nav>
+</header>
+    
+    <section class="mobilemain">
+      <article class="about-page" title="about">
+          <div class="main-content">
+            <h1 class="intro-title2 aboutintro2">cakey-cakey <br>by Marianna</h1>
+              <p class="about-paragraph" alt="explanation-paragraph">
+                Lorem ipsum dolor sit amet consectetur, adipisicing elit. Iste aliquam eaque, quae aspernatur accusamus voluptates repudiandae nemo vero praesentium quisquam placeat quos. Quis fugit illum commodi quasi, odit doloremque qui amet, deserunt ex minus labore. Ipsa voluptates expedita illum consectetur tempore veritatis ad? Asperiores quia magni in obcaecati suscipit eos neque quisquam corrupti necessitatibus. Labore quae molestiae totam. Incidunt, autem minima facere provident est earum?
+              </p>
```

Comment:
- [OPTIONAL] Just as I have suggested before, I think it will also be nice and make your project look more professional and presentable for your portfolio if you add real texts related to the event instead of using `Lorem ipsum` placeholder text. :+1: :smile: :wave: